app/livechat/client/views/app/tabbar/visitorForward.js

```diff
@@ -62,18 +67,40 @@ Template.visitorForward.events({
 		} else if (instance.find('#forwardDepartment').value) {
 			transferData.departmentId = instance.find('#forwardDepartment').value;
 		}
+		transferData.userLoggedIn = Meteor.userId();
+
+		// Check for settings whether forward is enabled or not.
+
+		if (settings.get('Livechat_ask_for_forward')) {
```

Comment:
Please, let's test if the setting is not activated, then execute the old code into this block and then `return`;
```suggestion
		if (!settings.get('Livechat_ask_for_forward')) {
    		     .....
    		     return;
    		}     
```